app/frontend/src/registry/components/people/Notes.vue

```diff
@@ -110,55 +192,127 @@ export default {
       }
       return typeMap[this.type]
     },
-    ...mapGetters(['userRoles'])
+    ...mapGetters(['userRoles', 'keycloak'])
   },
   methods: {
+    deleteNote() {
+      ApiService.delete(`${this.resourceType}/${this.guid}/notes`, this.activeNote.org_note_guid)
+      .then(() => {
+        this.activeNote = null;
+        this.alertText = "Note deleted.";
+        this.submitSuccess = true;
+        this.$emit('updated');
+      })
+    },
     noteSubmit () {
       // submit the note as a post request, triggered after confirming via popup
-      this.submitSuccess = false
-      this.submitError = false
-      this.submitLoading = true
+      this.submitSuccess = false;
+      this.submitError = false;
+      this.submitLoading = true;
       ApiService.post(`${this.resourceType}/${this.guid}/notes`, { note: this.noteInput })
-        .then((response) => {
-          const notes = this.$refs.noteSection
-
+        .then(() => {
+          const notes = this.$refs.noteSection;
           // Note submitted, set loading/success indicators and scroll down to the new note
           this.noteReset()
-          this.submitLoading = false
-          this.submitSuccess = true
-          smoothScroll(notes, 1000)
-          this.$emit('updated')
+          this.alertText = "Note added.";
+          this.submitLoading = false;
+          this.submitSuccess = true;
+          smoothScroll(notes, 1000);
+          this.$emit('updated');;
         }).catch((e) => {
-          this.submitLoading = false
-          this.submitError = e.response.data
+          this.submitLoading = false;
+          this.submitError = e.response.data;
```

Comment:
Might be good to have some type of feedback for the user in like `alertText` in case of any errors. For example, a note that's too long as the error `e.response.data.note[0]` "Ensure this field has no more than 2000 characters".

<img width="1018" alt="Screenshot 2024-03-28 at 12 35 03 PM" src="https://github.com/bcgov/gwells/assets/62899351/65d765dd-98ef-4f57-98b8-b05005f28a51">

Reply:
Added an alert since userbase for this section is closer to nil. But also added character count to text fields to disable ahead of time, preventing your use case from occuring 
![image](https://github.com/bcgov/gwells/assets/62873746/570d6898-cf54-463f-8995-b799bf439ab3)

Reply:
![image](https://github.com/bcgov/gwells/assets/62873746/b3e7c38b-fa28-4971-a8ca-03fc2b4584e9)